guides/release/accessibility/application-considerations.md

```diff
@@ -1,59 +1,59 @@
 
-> Both assistive technologies and conventional user agents can render text more accurately when the language of the Web page is identified. Screen readers can load the correct pronunciation rules. Visual browsers can display characters and scripts correctly. Media players can show captions correctly. As a result, users with disabilities will be better able to understand the content.
-> [WCAG Success Criterion 3.1.1: Intent](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html#intent)
+> Les outils d'accessibilités ainsi que les différents navigateurs WEB sont capables d'avoir un rendu plus précis quand la langue d'une page WEB est définie. Les lecteurs d'écrans peuvent ainsi affiner leurs prononciations. Les navigateurs WEB peuvent correctement afficher les caractères et les scripts. Les lecteurs vidéos peuvent afficher les bons sous-titres. Tout ceci permet aux utilisateurs, ayant un handicap, de mieux comprendre le contenu d'une page WEB.
+> [Critère de réussite du WCAG 3.1.1: Intention](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html#intent)
 
-A primary language should be defined on the `<html>` element's `lang` attribute. For new apps, you can use the `--lang` option on the `ember new` command to set the language for a new application.
+La langue principale doit être définie au niveau de l'attribut `lang` du tag `<html>`. Pour les nouvelles applications, vous pouvez utiliser l'option `--lang` de la commande `ember new`, afin de définir la langue pour votre nouvelle application.
 
 ```bash
 ember new mon-app --lang fr
 ```
 
-This command will create your application with French defined as the primary language
-on the `<html>` element's `lang` attribute.
+Cette commande va créer une application ayant le Français définit comme langue principale, sur l'attribut `lang` du tag `<html>`.
 
-For existing Ember apps, a developer may edit the `index.html` file or leverage [ember-intl](https://github.com/ember-intl/ember-intl).
+Pour une application existant, un développeur peut éditer le fichier `index.html` file ou installer l'addon [ember-intl](https://github.com/ember-intl/ember-intl).
 
-The `html` element may not have multiple `lang` _values_. If an element contains content in a language different from the primary, then you can provide the element its own `lang` attribute.
+L'attribut `lang` du tag `html` ne peut contenir plusieurs _valeurs_. Si un élément de la page contient du texte dans une langue différente que la langue principale, vous pouvez dans ce cas lui définir son propre attribut `lang`.
 
-![For example, the HTML tag may have a lang of "es" while a paragraph may have a lang of "en"](/images/accessibility/application-considerations/lang.png)
+![Par exemple, the tag HTML peut définir "es" en tant que langue principale et un paragraphe peut quant à lui avoir "en" en tant que langue définie](/images/accessibility/application-considerations/lang.png)
```

Comment:
@BlueCutOfficial @GreatWizard it looks like assets are missing

Reply:
Ooops, ok, we'll have a look 👍

Reply:
@KamiKillertO you can cherry-pick your work above `master`, now the assets are back in our repo 😃/😭